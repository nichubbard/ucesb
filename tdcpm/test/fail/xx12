/* ### Unit mismatch */

/* Fail due to wrong unit. */
r.e[2].m = 5;
