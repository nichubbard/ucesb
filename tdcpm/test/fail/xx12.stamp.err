0x5592cb665408 5.0000
Unit mismatch, cannot assign.
