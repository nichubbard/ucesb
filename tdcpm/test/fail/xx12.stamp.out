r.e(3).m = 5;
